src/test/java/com/ge/predix/metering/util/Constants.java

```diff
@@ -0,0 +1,34 @@
+package com.ge.predix.metering.util;
+
+public class Constants {
+	
+	public static final String SPACE_GUID = "space_guid";
+	public static final String SERVICE_PLAN_GUID = "service_plan_guid";
+	public static final String NAME = "name";
```

Comment:
This constants file combines both test and application constants. For those that are only used in the test I would move to the test class itself as they are not used in the application. I would only keep ones that are used in both test and application in this file. Even then, this class provides no context about these constants. So it might make sense to move them to constants in the specific class they are used in or to application properties.

Reply:
This constants file contains only test constants. The constants are used in NuregoIT and ServiceInstanceHelper.java, which are in two different packages. Should I make two separate constants files for each package?